spec/helpers/admin/application_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
```

Comment:
please remove all commented code like this

Reply:
removed.